cluster_experiments/power_config.py

```diff
@@ -254,6 +255,7 @@ def _raise_error_if_missing(self, attr, other_attr):
     "ttest_clustered": TTestClusteredAnalysis,
     "paired_ttest_clustered": PairedTTestClusteredAnalysis,
     "mlm": MLMExperimentAnalysis,
+    "delta": DeltaMethodAnalysis,
```

Comment:
also, you're adding it here but not adding the scale parameter in here. I think we need to add it if we want to init via config. I would also test init via config, I think it's missing

Reply:
I am not sure what you mean here. I get that it might be tricky to know that you have to pass an additional parameter or that the "scale" column has to appear somewhere. Is this what you are refering to?
If that is the case, do you have any recommendation on how to go around this? I am not sure I understand how to do it properly...

Reply:
It was pretty clear, thanks!
I think that now it should be working. I had to do some changes and raise a couple of additional errors to save guard. 
When time allows, could you check again and give more feedback?
Thanks!